### PR TITLE
Add Badges

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

- Adds badges throughout

This is be broken up nicely by commit to make the review burden easier.

**Note: this PR is to merge into the `piazz-badges` branch rather than master**

### Dragons

Nah

### How Has This Been Tested?

Tested locally

### CODE REVIEW

File: src/components/track/mobile/PlaylistTile.tsx [hunk 1]
```diff
@@ -103,6 +103,8 @@ const PlaylistTile = (props: PlaylistTileProps & ExtraProps) => {
     [styles.hide]: !artworkLoaded
   }
 
+  console.log({ ownerId: props.ownerId })
```
Comment (raymondjacobson) [inline src/components/track/mobile/PlaylistTile.tsx:106]: loggo doggo

File: src/components/track/mobile/PlaylistTile.tsx [hunk 2]
```diff
@@ -138,9 +140,12 @@ const PlaylistTile = (props: PlaylistTileProps & ExtraProps) => {
               <span className={cn(styles.userName, fadeIn)}>
                 {props.artistName}
               </span>
-              {props.artistIsVerified && (
-                <IconVerified className={styles.iconVerified} />
-              )}
+              <UserBadges
+                // userId={props.ownerId}
```
Comment (raymondjacobson) [inline src/components/track/mobile/PlaylistTile.tsx:144]: i imagine this was for test?

File: src/components/tracks-table/TracksTable.js [hunk 3]
```diff
@@ -79,6 +80,7 @@ const trackNameCell = (val, record, props) => {
 }
 
 const artistNameCell = (val, record, props) => {
+  console.log({ record })
```
Comment (raymondjacobson) [inline src/components/tracks-table/TracksTable.js:83]: loggo froggo

File: src/components/tracks-table/TracksTable.module.css [hunk 4]
```diff
@@ -101,13 +101,13 @@
 }
 
 :global(.ant-table-tbody > tr > td) .textCell {
-  white-space: nowrap; 
+  white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
-  position: absolute;
-  top: 0;
+  /* position: absolute; */
```
Comment (raymondjacobson) [inline src/components/tracks-table/TracksTable.module.css:107]: intentional?

File: src/containers/upload-page/components/FinishPage.js [hunk 5]
```diff
@@ -139,6 +139,7 @@ class FinishPage extends Component {
     const erroredTrackSet = new Set(erroredTracks)
     const isCreator = account.is_creator
 
+    console.log({ account })
```
Comment (raymondjacobson) [inline src/containers/upload-page/components/FinishPage.js:142]: loggo bloggo

File: src/containers/user-badges/UserBadges.tsx [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+  useSVGTiers = false
+}) => {
+  // Getting reselect selectors w/props to play nicely with useSelector
+  // is a bit of a song-and-dance:
+  // https://react-redux.js.org/next/api/hooks#useselector-examples
+  // const tierAndVerifiedSelector = useMemo(makeGetTierAndVerifiedForUser, [])
+
+  const tierAndVerifiedSelector = useMemo(makeGetTierAndVerifiedForUser, [])
+  const { tier, isVerified } = useSelector(state => {
+    return tierAndVerifiedSelector(state, { userId })
```
Comment (raymondjacobson) [inline src/containers/user-badges/UserBadges.tsx:54]: I think this could probably be nicer as 2 selectors, but not a strong pref

**Review by raymondjacobson — APPROVED** [review]
looks good! ill play around w/ the UI a bit more at some point to QA as well. I think we should pay special attention to what happens when there are super long display names

**Comment by audius-infra** [comment]
Preview this change https://frontend-demo.audius.co/piazz-client-balance-2

File: src/components/track/mobile/PlaylistTile.tsx (hunk 1 above)
Comment (piazzatron) [reply src/components/track/mobile/PlaylistTile.tsx:106]: wow i left a lot of logs

File: src/components/track/mobile/PlaylistTile.tsx (hunk 2 above)
Comment (piazzatron) [reply src/components/track/mobile/PlaylistTile.tsx:144]: indeed 😬

File: src/components/tracks-table/TracksTable.module.css (hunk 4 above)
Comment (piazzatron) [reply src/components/tracks-table/TracksTable.module.css:107]: yea

File: src/containers/user-badges/UserBadges.tsx (hunk 6 above)
Comment (piazzatron) [reply src/containers/user-badges/UserBadges.tsx:54]: Yea, one the fence on this one - gonna leave em as one I think

**Pull request merged**